static/src/javascripts/projects/common/modules/ui/first-pv-consent-banner.js

```diff
@@ -116,11 +117,8 @@ const trackInteraction = (interaction: string): void => {
 const canShow = (): Promise<boolean> =>
     Promise.resolve([hasUnsetAdChoices(), isInEU()].every(_ => _ === true));
 
-// TODO: enable the following checks, once page blocking behaviour has been approved by the business
 const canBlockThePage = (): boolean =>
-    [false, hasSeenTooManyPages(), isNotInHelpOrInfoPage()].every(
-        _ => _ === true
-    );
+    [isInTestVariant(), isNotInHelpOrInfoPage()].every(_ => _ === true);
```

Comment:
you probably want to unskip/edit the tests for this.

Reply:
good point!